src/lib/routes/admin-api/project/project-features.ts

```diff
@@ -194,7 +210,10 @@ export default class ProjectFeaturesController extends Controller {
                     requestBody: createRequestSchema(
                         'bulkToggleFeaturesSchema',
                     ),
-                    responses: { 405: emptyResponse },
+                    responses: {
+                        200: emptyResponse,
+                        ...getStandardResponses(400, 401, 403, 404, 413, 415),
```

Comment:
were you able to trigger all those codes in swagger UI? 
What about 409 that is in the code?